specification/azsadmin/resource-manager/update/Microsoft.Update.Admin/preview/2016-05-01/Updates.json

```diff
@@ -161,11 +161,6 @@
     "UpdateModel": {
       "description": "Properties of an update.",
       "properties": {
-        "dateAvailable": {
```

Comment:
removing properties is a breaking change, suggest to do it in new service tag

Reply:
The dateAvailable property was never implemented by the resource provider so this property was never populated.

Reply:
make sense if it's fixing swagger to align with service behavior.